src/spiceypy/spiceypy.py

```diff
@@ -14917,6 +14917,87 @@ def szpool(name: str) -> Tuple[int, bool]:
 ################################################################################
 # T
 
+@spice_error_check
+def tangpt(
```

Comment:
please add the argtypes for tangpt to libspicehelper.py https://github.com/AndrewAnnex/SpiceyPy/blob/c82a8720746f4ff80747f19d894071eee4f6607c/src/spiceypy/utils/libspicehelper.py#L2268